DSharpPlus.SlashCommands/SlashCommandsExtension.cs

```diff
@@ -827,12 +827,13 @@ private async Task<List<object>> ResolveInteractionCommandParameters(Interaction
                 var parameter = parameters.ElementAt(i);
 
                 //Accounts for optional arguments without values given
-                if (parameter.IsOptional && (options == null ||
-                                             (!options?.Any(x => x.Name == parameter.GetCustomAttribute<OptionAttribute>().Name.ToLower()) ?? true)))
+                if (parameter.IsOptional && (!options?.Any(x =>
+                        x.Name.Equals(parameter.GetCustomAttribute<OptionAttribute>().Name, StringComparison.InvariantCultureIgnoreCase)) ?? true))
```

Comment:
Here i removed `options == null` condition, as it's already taken care of in `options?.Any`